recipes/meshoptimizer/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "0.14":
+    url: "https://github.com/zeux/meshoptimizer/archive/v0.14.tar.gz"
+    sha256: "5a6e429ecf72325131b439a46297a690f0cab833c7563efdadab26c0a6cf8490"
+patches:
+  "0.14":
+    - patch_file: "patches/0.14/001_Fix build_on_old_AppleClang_versions.patch"
```

Comment:
Maybe name the patch "apple clang version fix"

Reply:
I used the title of the PR https://github.com/zeux/meshoptimizer/pull/130